app/presenters/tree_node/menu/item.rb

```diff
@@ -0,0 +1,36 @@
+module TreeNode
+  module Menu
+    class Item < TreeNode::Menu::Node
+      set_attribute(:key)      { "#{@options[:node_id_prefix]}__#{@object.feature}" }
+      set_attribute(:title)    { _(details[:name]) }
+      set_attribute(:tooltip)  { _(details[:description]) || _(details[:name]) }
+      set_attribute(:selected) { self_selected? || select_by_children_selected }
+
+      def children
```

Comment:
OK, please don't. 

I thought the whole point of this PR as opposed to https://github.com/ManageIQ/manageiq-ui-classic/pull/56 was that the tree node should never generate its own children.

So.. why this now? :)